docs/user/adding-new-metrics.md

```diff
@@ -48,7 +48,7 @@ category2.subcategory:  # Categories can contain subcategories
 
 The details of the metric parameters are described in [metric parameters](metric-parameters.md).
 
-The `metrics.yaml` file is used to generate `Kotlin` code that becomes the public API to access your application's metrics.
+The `metrics.yaml` file is used to generate `Kotlin` or `Swift` code that becomes the public API to access your application's metrics.
```

Comment:
Since we'll be adding more bindings soon-ish, I'll make this a bit more general. What about `... is used to generate code in the target language (e.g. Kotlin, Swift, ...) that becomes ...`